Students/Kamiko (A lazy Arisu cosplayer) - Syberian Orange and Arbiter1223, V0.27.student

```diff
@@ -0,0 +1,68 @@
+﻿Koukou Gurashi V0.27
+Soul Modified: 8/14/2018 10:06:54 PM
+------------------------------------------------------
+名|Kamiko|
+少女|True|
+ちんぽ|False|
+Hair(Front)|0|
+Hair(Back)|6|
+アホ毛|5|
+Hair(Tails)|2|
+HairPrimaryColour|33|
+HairSecondaryColour|19|
+目|17|
+口|3|
+肌|1|
+頭のアクセサリー|0|
+Eyewear|3|
+Pantsu|54|
+Sexual Preference|1|
+LeaderInfluence|-90|
+Snootiness|64|
+Gossips|27|
+Kindness|77|
+大胆|90|
+Intelligence|-6|
+Dexterity|69|
+ヤンデレ|False|
+Sanity|-18|
+ツンデレ|90|
+ラブラブ|84|
+StalksLover|True|
+FetishDominated|-16|
+HairTaste|4|
+Depression|6|
+LikesSchool|-1|
+Athletic|-852|
+LikesManga|-33|
+LikesAnime|-24|
+LikesBooks|13|
+LikesVideoGames|72|
+LikesMusic|86|
+変態|-20|
+LikesHumour|24|
+LikesOccult|-54|
+生|True|
+クーデレ|False|
+忠実|True|
+声|1|
+Lingo Great|sick-ass|
+Lingo Crappy|bad|
+Class1_Proficiency|0|
+Class2_Proficiency|0|
+Class3_Proficiency|255|
+Class4_Proficiency|0|
+Class5_Proficiency|0|
+Class6_Proficiency|0|
+Class7_Proficiency|0|
+Class8_Proficiency|0|
+Accent|0|
+Energy|20|
+ThighHigh Base|23|
+ThighHigh Extra|0|
+骨盤&太もも|2|
+声ピッチ|3|
+頭のアクセサリーの色|1|
+おっぱい|2|
+Lingo Very|14|
+(╯°□°）╯You ruined it!! You killed the magic!!
```

Comment:
Everything else looks okay. Just fix those two things, and she'll be good to go. :D